src/themes/dark.scss

```diff
@@ -98,6 +98,8 @@
   --wxc-select--focus--color: #{$wxc-white-alpha-95};
   --wxc-select--focus--border: 1px solid #{$wxc-blue-60};
 
+  --wxc-loading-logo: url(../../docs/assets/images/loading-dark.gif);
```

Comment:
put this at the top, just after meeting-background

Reply:
is it possible to put the file in the themes folder? eg: `src/themes/dark/loading.gif`